boards/arm/96b_carbon_nrf51/96b_carbon_nrf51.yaml

```diff
@@ -9,3 +9,6 @@ toolchain:
   - gccarmemb
 supported:
   - ble
+testing:
+  ignore_tags:
+    - net
```

Comment:
mea culpa, this specific board does have 32kb of RAM, so I guess we have more flexibility here. microbit however fails on 20 tests when the net tag is unignored...